docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -557,6 +557,23 @@ spec:
 > **Note:** The IPv6 address for the VM and default gateway **must** be the ones
 > shown above.
 
+### masquerade - IPv6 single-stack support
+
+`masquerade` mode can be used in IPv6 single stack clusters to provide a VM
+with an IPv6 only connectivity.
+
+As with the IPv4 `masquerade` mode, the VM can be contacted using the pod's IP
+address - which will be in this case the IPv6 one.
+Outgoing traffic is also "NAT'ed" to the pod's respective IPv6 address.
+
+As with the dual-stack cluster, the configuration of the IPv6 address and the default route is
+not automatic; it should be configured via cloud init, as shown in the dual-stack section.
+
+Unlike the dual-stack cluster, which has a DHCP server for IPv4, the IPv6 single stack cluster
+has no DHCP server at all. Therefore, the VM won't have the search domains information and
+reaching a destination using its FQDN is not possible.
+Tracking issue - https://github.com/kubevirt/kubevirt/issues/7184
```

Comment:
Can this be workaround manually?

Reply:
Yes, a user can copy the /etc/resolv.conf and /etc/hosts from the virt-launcher pod. But I don't want to document it since it is an ugly workaround that includes too many implementation details.

Reply:
Ok. I thought we can just mention in a note that one can define it manually based on the information in the pod.